spec.html

```diff
@@ -26731,13 +26751,13 @@ <h1>Cyclic Module Records</h1>
             </tr>
             <tr>
               <td>
-                [[AsyncEvaluation]]
+                [[AsyncEvaluationOrder]]
               </td>
               <td>
-                a Boolean
+                ~unset~, an integer, or ~done~
               </td>
               <td>
-                Whether this module is either itself asynchronous or has an asynchronous dependency. Note: The order in which this field is set is used to order queued executions, see <emu-xref href="#sec-async-module-execution-fulfilled"></emu-xref>.
+                This field, only used on modules that are either themselves asynchronous or have an asynchronous dependency, determines the order in which execution of pending modules is queued. See <emu-xref href="#sec-async-module-execution-fulfilled"></emu-xref>.
```

Comment:
I think this description can be improved.

> only used on modules that are either themselves asynchronous or have an asynchronous dependency

So what value does it take for modules that do not meet these criteria? Also, you explain what the integer value is used for but not what the other spec enum values are supposed to mean or when they are used.

Here's a start

```suggestion
                This field, for modules that are either themselves asynchronous or have an asynchronous dependency, determines the order in which execution of pending modules is queued by <emu-xref href="#sec-async-module-execution-fulfilled"></emu-xref>. After evaluation, it is set to ~done~. For other modules, the value is ~unset~.
```

Also I think I'd prefer we use a more descriptive enum name than `~unset~`.

Reply:
What do you think about `~sync~` instead of `~unset~`?

Reply:
SGTM

Reply:
I think it's confusing to have async modules start out as `~sync~`; I was happier with `~unset~`.

Reply:
I see that this is now marked as `[editor call]` -- pick one and then tell me :)

Reply:
@syg and I found `~sync~` confusing; let's put it back to `~unset~` (or we can do it when landing). Sorry for the churn.